frontend/cypress/integration/07-account_creation_spec.js

```diff
@@ -56,163 +56,154 @@ Cypress.Commands.add("verifySecurityCode", (code) => {
   cy.get(submitButton).first().click();
 });
 
-describe(
-  "Okta account creation",
-  {
-    retries: {
-      runMode: 2,
-      openMode: 1,
-    },
```

Comment:
The diff here is a little wild, but I assume the only thing that actually changed was moving this retry logic to the cypress.json file?

Reply:
Yeah, so previously I had configured only that spec to do retries - moving it to `cypress.json` make it apply globally to all specs!

Reply:
Perfect, thanks! Just wanted to double-check that there weren't any other logic changes in the account creation test 👍